src/ui/common/src/components/layouts/menuSidebar.tsx

```diff
@@ -140,6 +141,10 @@ const MenuSidebar: React.FC<{
     fetchVersionNumber();
   }, [user.apiKey]);
 
+  const { data, error, isLoading } = useVersionNumberGetQuery({
```

Comment:
Remove, this didn't quite work out for some reason, need to dig through the RTK docs some more and learn about how to send request with no params, and just the API key header.